src/shacl2code/lang/templates/python.j2

```diff
@@ -1943,13 +1943,13 @@ CONTEXT_URLS = [
 {%- if not class.is_abstract %}
 @register("{{ class._id }}"{%- if context.compact(class._id) != class._id %}, "{{ context.compact(class._id) }}"{%- endif %})
 {%- endif %}
-class {{ varname(class.clsname) }}(
+class {{ varname(*class.clsname) }}(
```

Comment:
What is the `*` for?

Reply:
`*` in python "expands" a tuple into function arguments, so for example, if `class.clsname` is `['a', 'b', 'c']` this is equivalent to `varname('a', 'b', 'c')`

Reply:
oh cool. so jinja just does that. OK, +1.